pkg/edit/vars.go

```diff
@@ -41,9 +41,8 @@ func initVarsAPI(ed *Editor, nb eval.NsBuilder) {
 // ▶ (num 2)
 // ```
 //
-// Note that if you pass a variable as the `$init` argument, `edit:add-var`
-// doesn't add the variable "itself" to the REPL namespace, it creates a new
-// variable, only using the existing variable's value:
+// Note that `edit:add-var` doesn't add (or alias) the `$init` variable "itself" to the REPL
```

Comment:
I'm not sure why you removed the "if you pass variable ..." wording. This note is really only relevant when passing a variable to `$init`. "Aliasing" is not a concern when you do something like `edit:add-var foo foo`. You can't alias a value since values themselves are immutable.

Reply:
The implication of the original statement is that the `$init` argument is optional which isn't true. The problem here is the existing command notation doesn't readily lend itself, to a casual reader, to distinguishing between a value and a reference to a variable. In fact, you can't pass a "variable" as the initial value to this command -- you can only pass a value (possibly derived from an existing variable). That is the source of my (and no doubt other's) misunderstanding despite the otherwise clear syntax and what caused me to mistakenly think `edit:add-var umask $unix:umask` would work.

Reply:
> The implication of the original statement is that the `$init` argument is optional which isn't true.

I'm still not quite sure how that misunderstanding arises. Is it because the wording "if you pass" can lead the reader to think that they don't have to pass anything at all as `$init`?

Reply:
> Is it because the wording "if you pass" can lead the reader to think that they don't have to pass anything at all as $init? 

Yes, and where I wrote "a value" in my previous comment I meant a literal string, number, etc. as opposed to a variable deref.